docs/Tutorial.md

```diff
@@ -47,7 +47,7 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import { App } from './App';
 
-ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
+ReactDOM.createRoot(document.getElementById('root')!).render(
```

Comment:
Don't know how others feel about this change, but I liked to original code better personally

Reply:
Yet we're based on Vite, and they did the same in https://github.com/vitejs/vite/commit/6fe14911deac34f5874687705a46deb9caf1cb10

Reply:
alright, valid point :grin: